src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -6,13 +6,12 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>READYTORUN;$(DefineConstants)</DefineConstants>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
If you do `git clean -dfx` to delete any user config, open the crossgen2.sln solution in VS and try to compile something with F5, does that work? I would assume it would crash because AnyCPU tends to create a 32bit process and we won't be able to load RyuJIT.

The `Platforms` was here so that VS doesn't try to create AnyCPU configs that don't really work.

Reply:
Hmm... This is causing Linux arm/arm64 scenarios to fail. I think we never actually noticed it because we were never checking the return code of dotnet publish.
Let me try to add arm and arm64 for Platform and see if that fixes the issue. Or explore passing a PlatformTarget value from the dotnet publish command line

Reply:
Adding arm and arm64 didn't work with the publish command. I'm experimenting with this instead:
`    <PlatformTargets Condition="'$(BuildArch)' != ''">$(BuildArch)</PlatformTargets>`

Reply:
Passing a value PlatformTargets from the dotnet publish command worked, so removing AnyCPU